README.md

```diff
@@ -68,13 +68,12 @@ target = NumericalTarget(
 )
 objective = SingleTargetObjective(target=target)
 ```
-
-In cases where we need to consider multiple (potentially competing) targets, the
-role of the `Objective` is to define additional settings, e.g. how these targets should
-be balanced.
-In `SINGLE` mode, however, there are no additional settings.
-For more details, see 
-[the objective section of the user guide](https://emdgroup.github.io/baybe/userguide/objective.html).
+In cases where we are confronted with multiple (potentially conflicting) targets,
+the `DesirabilityObjective` can be used instead. It allows to define additional
+settings, such as how these targets should be balanced.
+For more details, see the
+[objective section](https://emdgroup.github.io/baybe/userguide/objectives.html)

```

Comment:
@AVHopp: I guess it's due to this change here, since it's currently pointing to a not-yet-existing page. We've had that problem in the past and that was one of the reasons for the force option, but I can't remember what was the final decision how to handle it and why it no longer works even though we encountered that situation already before

Reply:
Guess we can resolve since we agreed how to proceed with this point?

Reply:
Yes, will resolve